libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/ModelOverview.tsx

```diff
@@ -318,6 +338,32 @@ export class ModelOverview extends React.Component<
                   .helpMeChooseMetricsButton
               }
             </ActionButton>
+            <ComboBox
```

Comment:
these should only be added for object detection and not other tasks - it looks like these are added for all possible tasks which is incorrect I think?

Reply:
Yes I had the same question above: how do we only surface this for the OD scenario?

Reply:
we use && on boolean variables.  EG, see:
https://github.com/microsoft/responsible-ai-toolbox/blob/main/libs/model-assessment/src/lib/ModelAssessmentDashboard/ModelAssessmentDashboard.tsx#L147

```
          {this.state.saveCohortVisible && (
            <SaveCohort
              isOpen={this.state.saveCohortVisible}
              onDismiss={(): void =>
                this.setState({ saveCohortVisible: false })
              }
              onSave={this.onSaveCohort}
              temporaryCohort={this.state.selectedCohort}
              baseCohort={this.state.baseCohort}
            />
          )}
```